docs-src/concepts/what-is-a-history-service.md

```diff
@@ -7,11 +7,13 @@ tags:
   - term
 ---
 
-The History Service is responsible for persisting Workflow Execution state and determining what to do next to progress the Workflow Execution by using [History Shards](/concepts/what-is-a-history-shard).
+The History Service is responsible for persisting Workflow Execution state to the Workflow History.
+When the Workflow Execution is able to progress, the History Service adds a Task with the Workflow's updated history to the Task Queue.
+From there, a Worker can poll for work, receive this updated history, and resume execution.
 
 ![History Service](/diagrams/temporal-history-service.svg)
 
-The total number of History Services can be between 1 and the total number of History Shards.
+The total number of History Services can be between 1 and the total number of [History Shards](/concepts/what-is-a-history-shard).
```

Comment:
I would use the term "History Service process". The number of processes can be larger than the number of shards, it's just that those beyond the shard count would not have any useful work to do. It's also highly unlikely to need a History Service process per shard. As an example, we normally recommend one History Service process per 500 shards.